tableauserverclient/server/request_options.py

```diff
@@ -25,6 +28,35 @@ class Direction:
         Desc = 'desc'
         Asc = 'asc'
 
+    class Builder:
+        def __init__(self):
+            self.__pagenumber = 1
```

Comment:
I'm not sure the `__` is necessary (it invokes name mangling to avoid inheritance problems but here I think you can just make it the `_`... it's more 'polite' form :P